CIP-0013/CIP-0013.md

```diff
@@ -100,17 +111,27 @@ For brevity, essential in many Internet contexts, `poolticker`  must be supporte
 
 When there is more than one pool registered with any of the specified `poolticker` parameters (whether for pool groups which have the same ticker for all pools, or for separate pools using the same ticker), the choice to which pool(s) to finally delegate is left to the user through the wallet UI.
 
-### Transaction balancing and signing URI queries
-
-TBD
-
 #### Interpretation of `proportion`:
 
 * If only one stake pool is specified, any proportion is meaningless and ignored.
 * If all stake pools have a numerical proportion, each component of the resulting stake distribution will have the same ratio as the provided `proportion` to the sum of all the propotions.
 * Any missing `proportion` is assigned a precise value of `1`.
 * If a stake pool is listed multiple times, the URI is rejected as invalid.
 
+### Transaction balancing and signing URI queries
+
+The JSON payload of this URI shall be passed by Daedalus, or other wallets that handle the link format, directly to the [CWB balance transaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/balanceTransaction) API as its request body.
+
+The return value of that endpoint shall then be sent to the [Sign transaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/signTransaction) API.
+
+Finally, the signed transaction shall be sent to the [Submit transaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postExternalTransaction) API.
+
+#### Interpretation of `payload`:
+
+The "payload" parameter is the JSON representation of an [ExportTx](https://github.com/input-output-hk/plutus/blob/master/plutus-contract/src/Plutus/Contract/Wallet.hs#L113-L120) value, gzipped and Base-64 encoded using the URL-safe alphabet
```

Comment:
@j-mueller The referenced datatype contains a [`signatories `](https://github.com/input-output-hk/plutus/blob/5e9d0d9046b0ca7e3cfbef99661f67599122d5a2/plutus-contract/src/Plutus/Contract/Wallet.hs#L118) field, but I believe Matthias just confirmed with you that such a field would be un-needed? So I suppose it should be updated.

Reply:
I'm currently updating the export code on our side, I'll add a full example to this PR and remove the link

Reply:
Thanks! Please keep us posted 🙏

Reply:
@anviking changed, let me know if you need more!

Reply:
@Anviking are you ok with this now?

Reply:
> I'll add a full example to this PR and remove the link

@j-mueller please let us know once you sort this out 🙏